src/Examples.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Link, RouteComponentProps } from 'react-router-dom';
+
+import * as Snippet from "./Snippet";
+
+import * as Simple from "./examples/Simple";
+
+interface IExample {
+    lang: string;
+    code: string;
+}
+
+const sources = {
```

Comment:
I think we should either move this to routing or make it clearer that the routes `/example/:name` are based on these `sources` names. It was a bit hard for me to make the connection, but maybe that's just me.

Reply:
Strong disagree, I don't think Routing should care about what examples we have, seems like unnecessary coupling.

Reply:
Hmmm I agree, it would be better to stay here. It's just non-obvious to me that this is where all the example names are kept. Would it be better to let examples specify their own name instead?

Reply:
@jaspercb thoughts?

Reply:
Oh how about we maintain a file called ExampleNames.tsx in `examples/` that `export default`s the equivalent of `sources`? That way we keep the naming close to the examples themselves. We can then import that into this `Examples.tsx`.

Reply:
I don't like that because then there's a file in `examples/` that isn't the same "type" as all the other files.  Can we have ExampleNames.tsx but keep it outside `examples/`?

Reply:
ah true true. Keeping it outside SGTM.